scripts/visualize/visualize.map_geometry.R

```diff
@@ -0,0 +1,13 @@
+visualize.map_geometry <- function(viz){
+  
+  deps <- readDepends(viz)
+  geom_feature <- deps[['geom_feature']]
+  geom_context <- deps[['geom_context']]
+  
+  library(sp)
+  
+  png(viz[['location']])
+  plot(geom_context, col="grey")
+  plot(geom_feature, col="lightyellow")
```

Comment:
don't need an `add=TRUE` here?

Reply:
oh yep

Reply:
these aren't totally perfect - need to work on limits and stuff. Just wanted something up and running.

Reply:
👍  go ahead an merge if you are ready